docs/features/pm2-api.md

```diff
@@ -36,22 +34,120 @@ pm2.connect(function(err) {
+  * `cwd` - The working directory to start the process with.
+  * `error` - (Default: "~/.pm2/logs/app_name.err") The path to a file to append stderr output to.
+  * `output` - (Default: "~/.pm2/logs/out.log") The path to a file to append stdout output to. 
+  * `log_file` - Specifies a file path to append both stderr and stdout output. 
+  * `log_date_format` - The display format for log timestamps (eg "YYYY-MM-DD HH:mm Z"). The format is a [moment display format](http://momentjs.com/docs/#/displaying/).
+  * `pid` - (Default: "~/.pm2/logs/~/.pm2/pids/app_name-id.pid") The path to a file to write the pid of the started process. The file will be overwritten. Note that the file is not used in any way by pm2 and so the user is free to manipulate or remove that file at any time. The file will be deleted when the process is stopped or the daemon killed.
+  * `min_uptime` - The minimum uptime of the script before it's considered successfully started. 
+  * `max_restarts` - The maximum number of times in a row a script will be restarted if it exits in less than `min_uptime`.
+  * `max_memory_restart` - If set and `script`'s memory usage goes about the configured number, pm2 restarts the `script`. Uses human-friendly suffixes: 'K' for kilobytes, 'M' for megabytes, 'G' for gigabytes', etc. Eg "150M".
+  * `kill_timeout` - (Default: 1600) The number of milliseconds to wait after a `stop` or `restart` command issues a `SIGINT` signal to kill the script forceably with a `SIGKILL` signal. 
+  * `restart_delay` - (Default: 0) Number of millseconds to wait before restarting a script that has exited.  
+  * `interpreter` - (Default: "node") The interpreter for your script (eg "python", "ruby", "bash", etc). The value "none" will execute the 'script' as a binary executable.
+  * `exec_mode` - (Default: 'fork') If set to 'cluster', will enable clustering (running multiple instances of the `script`). [See here for more details](http://pm2.keymetrics.io/docs/usage/cluster-mode/).
+  * `instances` - (*Default: 1*) How many instances of `script` to create. Only relevant in `exec_mode` 'cluster'.
+  * `mergeLogs` - (Default: false) If true, merges the log files for all instances of `script` into one stderr log and one stdout log. Only applies in 'cluster' mode. For example, if you have 4 instances of 'test.js' started via pm2, normally you would have 4 stdout log files and 4 stderr log files, but with this option set to true you would only have one stdout file and one stderr file.
+  * `watch` - If set to `true`, the application will be restarted on change of the `script` file.
+  * `force` (Default: false) By default, pm2 will only start a script if that script isn't already running. If `force` is set to true, pm2 will start a new instance of that script.
+  * `cron`
+  * `runAsUser`
+  * `runAsGroup`
```

Comment:
runAsUser and runAsGroup have been deprecated, you can remove them to avoid confusion ;)